bytes/src/main/scala/io/iohk/ethereum/utils/ByteStringUtils.scala

```diff
@@ -61,7 +61,7 @@ object ByteStringUtils {
 
   def concatByteStrings(elements: Iterator[ByteStringElement]): ByteString = {
     val builder = new mutable.ArrayBuilder.ofByte
-    elements.foreach(el => builder.addAll(el.asByteArray))
+    elements.foreach(el => builder ++= el.asByteArray)
```

Comment:
looks like an unrelated change. intentional?

Reply:
This change was necessary because of cross publishing. We want to use these libraries from Scala 2.12 as well as Scala 2.13, and 2.12 doesn't have `addAll`.